telemetry/definitions/commonDefinitions.json

```
@@ -1448,6 +1448,11 @@
             ],
             "description": "User-provided policy type of the reference file. Applicable to only CheckNoNewAccess check type."
         },
+        {
+            "name": "registrationExpiresAt",
```

should this have a "sso" prefix like the other field added in this pr?
```suggestion
            "name": "ssoRegistrationExpiresAt",
```

---

Oops, this was intended, good catch